pkg/apiserver/logsearch/service.go

```diff
@@ -84,6 +84,11 @@ type TaskGroupResponse struct {
 	Tasks     []*TaskModel   `json:"tasks"`
 }
 
+type TaskGroupStats struct {
+	TaskGroup   TaskGroupModel                `json:"task_group"`
```

Comment:
You need to change the `TaskGroup` model to include the number, otherwise all `Task`s are still being queried and it is very inefficient: `err = s.db.Where("task_group_id = ?", taskGroup.ID).Find(&tasks).Error`.

Reply:
Oh I know what your mean. `RequestTargetStatistics` should be persisted into DB, ike `TaskGroupModel` in profiling module.

I will do this very soon.